src/pageObject.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return `Conduit logo`;
+  }
 };
 
 class PageObject {
+  constructor(url, header) {
+    this.url = url;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, header, slug) {
```

Comment:
"constructor taskes `url `and `slug`. Constructor should modify original `url` by adding `slug` at the end"

Reply:
Changes are done